docs/two-phase-commit-transactions.md

```diff
@@ -286,7 +286,10 @@ TwoPhaseCommitTransaction tx = transactionManager.join("<TRANSACTION_ID>");
 ...
 
 // Resume the transaction by using the transaction ID.
-TwoPhaseCommitTransaction tx1 = transactionManager.resume("<TRANSACTION_ID>")
+TwoPhaseCommitTransaction tx1 = transactionManager.resume("<TRANSACTION_ID>");
+
+// Or you can re-join the transaction by using the transaction ID.
+TwoPhaseCommitTransaction tx2 = transactionManager.join("<TRANSACTION_ID>");
 ```
 
 {% capture notice--info %}
```

Comment:
To avoid having two note sections stacked on top of each other, how about we combine these two notes as follows? (Sorry, I can't select the first note in this suggestion because it's out of scope of the changes.):

````suggestion
{% capture notice--info %}
**Note**
To get the transaction ID with `getId()`, you can specify the following:

```java
tx.getId();
```

In addition, when using `join()` to re-join a transaction, if you have not joined the transaction before, a new transaction object will be returned. On the other hand, when using `resume()` to resume a transaction, if you have not joined the transaction before, `TransactionNotFoundException` will be thrown.
{% endcapture %}

<div class="notice--info">{{ notice--info | markdownify }}</div>
````

Reply:
Thank you for your suggestion. I've fixed in 0172b5830aa9e306ba2cc7726fccec24201eab7e. Please take a look when you have time!